### ISSUE TITLE
brew install node followed by npm install -g browserify doesn't create /usr/local/bin/browserify

### ISSUE THREAD

**Issue by sglaser** [Homebrew/homebrew#47449]
If I install node using the current homebrew formula on osx el capitan, it correctly installs node v5.3.0.

If I then try to install the browserify node package using `sudo npm install -g browserify` the package installs correctly but there is nothing left in `/usr/local/bin/browserify`

If I install node using the package from nodejs.org and then install browserify using npm, this works properly.

I suspect that this may be due to the browserify package.json file being a bit unusual. Specifically it has:

`"bin": {
"browserify": "bin/cmd.js"
},`

instead of the more typical

`"bin": {
"browserify": "bin/browserify"
},`

See also https://github.com/substack/node-browserify/issues/1465

**Comment by bfontaine** [comment]
Using Node from Homebrew and `npm install -g browserify` (note I don’t use `sudo`) I get `browserify` as expected:
```
$ which browserify
/usr/local/bin/browserify
```

**Comment by DomT4** [comment]
Same result here as @bfontaine.
```
~> npm install -g browserify
/usr/local/bin/browserify -> /usr/local/lib/node_modules/browserify/bin/cmd.js
/usr/local/lib
└─┬ browserify@12.0.1
```
And so on.

**Comment by sglaser** [comment]
I'll try it again later today.

My version of npm was installing everything into something like /usr/local/lib/Cellar/node*/node_modules

Not sure why and I'm not at the computer right now. 

Steveg

**Comment by DomT4** [comment]
Let us know if we can help further here, Thanks!

You may have an old `npmrc` or something pointing at the Cellar perhaps, if you didn't manage to resolve this locally.

**Comment by sglaser** [comment]
Just reinstalled from scratch using home-brew and everything installs as it should.

I’m not sure why npm was putting things in /usr/local/lib/Cellar/node*/node_modules instead of /usr/local/lib/node_modules.

I did have both node flavors installed at different times (nodes.org vs brew). It’s quire likely that I didn’t fully remove one before switching. I also defaulted .npmrc to global at one point (bad idea, but I was learning).

Steve Glaser
sglaser@sglaser.com
